lib/CirclesManager.php

```diff
@@ -206,6 +207,40 @@ public function startAppSession(string $appId, int $appSerial = Member::APP_DEFA
 		$this->federatedUserService->setLocalCurrentApp($appId, $appSerial);
 	}
 
+	
+	/**
+	 * Run a single method as Super, then Super level will be removed
+	 *
+	 * @throws Throwable
+	 * @noinspection PhpUndefinedVariableInspection
```

Comment:
This sounds scary, undefined variables will throw errors in PHP 8.2

Reply:
Returned is mixed
let's put this in standby, merge is not necessary.